rosette/cmaketest.sh

```diff
@@ -1,35 +1,60 @@
 #!/usr/bin/env bash
 
-
-
-# Create file for printing results
-touch rbl/rosette/results.txt
+# Create/clear file for printing results
+RESULTS=rbl/rosette/results.txt
+cat > "${RESULTS}" < /dev/null
 
 # Build Rosette
 ./clean.sh
 ./build.sh
+if [ $? -ne 0 ]
```

Comment:
I would put `set -e` at the top of this file and then you don't need this, because the script will terminate on first command that returns non-zero exit code.